esx_service/vmdk_ops.py

```diff
@@ -1786,11 +1826,16 @@ def handleVmciRequests(port):
     cartel = c_int32()
     vmci_grab_listening_socket(port)
 
-
     while True:
+        # Check the STOP flag - exit the loop if the flag is set
+        if stopEvent.isSet():
+            logging.info("Received stop request - stop processing new request")
+            break;
+
+        # Listening on VMCI socket
+        logging.info("lib.vmci_get_one_op: waiting for new request...")
```

Comment:
There is no need for info here. We used to have it and it polluted the logs. If you need it for debug , feel free to use log.debug.

Reply:
Will change it to debug - it's very useful for debugging.